moderation/src/main/java/pl/fratik/moderation/commands/RolaCommand.java

```diff
@@ -53,11 +59,26 @@ public boolean execute(@NotNull CommandContext context) {
         }
         Role rola = (Role) context.getArgs()[0];
         GuildConfig gc = guildDao.get(context.getGuild());
+
         if (!gc.getUzytkownicyMogaNadacSobieTeRange().contains(rola.getId())) {
             context.send(context.getTranslated("rola.not.defined"));
             return false;
         }
         try {
+            Integer maxRolesSize = gc.getMaxRoliDoZamododania();
+            if (maxRolesSize == null) maxRolesSize = 100;
+
+            if (maxRolesSize == 0) {
+                context.send(context.getTranslated("rola.disabled"));
```

Comment:
uh nie, jakby ktoś chciał wyłączyć fb!rola zrobiłby to przez wyłącz komendę/nie ustawił żadnych ról do samonadania
jeżeli maxRolesSize == 0 to powinno to działać jako "możesz sobie przydzielić ile chcesz"